src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1310,11 +1310,10 @@ impl_types layout_optimizer::get_preferred_impl_type(program_node& node, format
             bool first_conv = input_layout.size.feature[0] <= 4;
             bool enable_onednn_dw_fp16_conv = fp16_input && is_depthwise;
             if (((has_groups && !enable_onednn_dw_fp16_conv) || first_conv) &&
-                (output_layout.format == format::b_fs_yx_fsv16 || output_layout.format == format::bs_fs_yx_bsv32_fsv16) &&
+                (output_layout.format == format::b_fs_yx_fsv16 || output_layout.format == format::bs_fs_yx_bsv32_fsv16 ||
+                output_layout.format == format::b_fs_yx_fsv32) &&
```

Comment:
I think this should be like below. 
```suggestion
                output_layout.format == format::b_fs_yx_fsv16 &&
```

Reply:
This fix will be applied by other PR(Mingyu's one).